__tests__/app.test.js

```diff
@@ -140,3 +138,46 @@ describe("Get /api/articles", () => {
         })
     })
 })
+
+describe("GET /api/articles/:article_id/comments", () => {
+    test("200: Should respond with an array of comments for the given article_id with the relevant properties, ordered by most recent comment first", () => {
+        return request(app)
+        .get("/api/articles/1/comments")
+        .expect(200)
+        .then(({ body }) => {
+                for (let i = 0; i < body.comments.length; i++){
+                expect(body.comments[i].article_id).toBe(1)
+                expect(body.comments.length).toBeGreaterThan(0)
+            }
+            body.comments.forEach((item) => {
+                expect(item).toMatchObject({
+                    comment_id: expect.any(Number),
```

Comment:
Could we add the article_id column to this list of expects too and assert that it should be 1.